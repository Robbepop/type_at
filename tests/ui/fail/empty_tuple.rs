use tuple_type::TypeAt;

fn main() {
    let _: _  = <() as TypeAt<0>>::Type::default();
}
