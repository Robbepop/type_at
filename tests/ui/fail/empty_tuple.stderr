error[E0277]: the trait bound `(): TypeAt<0_usize>` is not satisfied
 --> $DIR/empty_tuple.rs:4:17
  |
4 |     let _: _  = <() as TypeAt<0>>::Type::default();
  |                 ^^^^^^^^^^^^^^^^^^^^^^^ the trait `TypeAt<0_usize>` is not implemented for `()`
